crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -788,6 +788,7 @@ def proposal_status
           ).first
         ]
       end
+      result.compact!
```

Comment:
It will be empty in the case of the bug I fixed, but I don't believe `compact` does what you think:

```
irb(main):001:0> [].compact
=> []
```